docs/transpile/representing_quantum_computers.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Representing quantum computers
+description: The necessary backend information for successfully transpiling quantum circuits
+
+---
+
+# Representing quantum computers
+
+To construct an equivalent circuit that can run on a specific backend, the transpiler needs certain details about the backend.  Typically, this information is found in the `backend` or `target = backend.target` class, so you don't need to pass anything further to the transpiler. (TODO: check with Matthew about how this is moving, seems there is no `target` attribute for older `Backend` classes). However, if more infomation is provided, the transpiler can use it to try to produce the best circuit to run on that hardware.  
+
+<Admonition type="note">
+Because many of the underlying transpilation algorithms are stochastic, there is no guarantee that a better circuit will be found. 
+</Admonition>
+
+<span id="default-config"></span>
+## Default configuration
+
+The simplest use of the transpiler is to provide all the backend infomation by providing the `backend` or `target` (TODO: only available on newer backends?). To better understand how the transpiler works, construct a circuit and transpile it with different information:
+
+Import the necessary libraries and instantiate the backend:
+
+```python
+    from qiskit import transpile
+    from qiskit.circuit.library import EfficientSU2
+    from qiskit.providers.fake_provider import FakeSherbrooke
+
+    backend = FakeSherbrooke()
+    target = backend.target
+```
+
+The EfficientSU2 circuit consists of layers of single qubit operations spanned by SU(2) and `CX` entanglements. This is a heuristic pattern that can be used to prepare trial wave functions for variational quantum algorithms or classification circuits for machine learning.
+
+```python
+    qc = EfficientSU2(12, entanglement='circular', reps=1)
+    qc.decompose(reps=1).draw('mpl', style='iqp')
+```
+![The 12-qubit test circuit](/images/transpile/representing_quantum_computers/qc-circular.png)
+
+Transpile the circuit with the simplest options. This example uses `optimization_level=2` to highlight a feature:
```

Comment:
Was "transpile the circuit with the simplest options" meant to be a heading? Feels a bit weird grammatically to start the paragraph with that sentence

Reply:
"Fewest options (kwargs)" and I removed `optimization_level=2` since the default level 1 actually works for the example.

Reply:
Not seeing @nbronn 's change here.  Trying to figure out why.